src/Mod/Sketcher/Gui/SketcherSettings.ui

```diff
@@ -163,6 +163,56 @@ Requires to re-enter edit mode to take effect.</string>
     </widget>
    </item>
    <item row="3" column="0">
+    <widget class="QGroupBox" name="groupBox_6">
+     <property name="sizePolicy">
+      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
+       <horstretch>0</horstretch>
+       <verstretch>0</verstretch>
+      </sizepolicy>
+     </property>
+     <property name="title">
+      <string>Contextual Constrain</string>
+     </property>
+     <layout class="QGridLayout" name="gridLayout_general">
+      <item row="0" column="0">
+       <widget class="QLabel" name="dimensioningLabel">
+        <property name="text">
+         <string>Dimensioning constraints:</string>
+        </property>
+       </widget>
+      </item>
+	  <item row="0" column="1">
+       <widget class="QComboBox" name="dimensioningMode">
+        <property name="toolTip">
+         <string>Selection the type of dimensioning constraints for your toolbar:
```

Comment:
Sounds wrong 'Selection the type..' maybe 'Select the type...'

Reply:
Thanks !